riff-raff/app/notification/hooks.scala

```diff
@@ -85,17 +85,18 @@ object HookConfig {
     HookConfig(UUID.randomUUID(), projectName, stage, url, enabled, new DateTime(), updatedBy)
 }
 
-object HooksClient extends LifecycleWithoutApp with Logging {
-  trait Event
-  case class Finished(uuid: UUID, params: DeployParameters)
-
+class HooksClient(wsClient: WSClient) extends Lifecycle with Logging {
   lazy val system = ActorSystem("notify")
   val actor = try {
-    Some(system.actorOf(Props[HooksClient], "hook-client"))
-  } catch { case t:Throwable => None }
+    Some(system.actorOf(Props(classOf[HooksClientActor], wsClient), "hook-client"))
```

Comment:
A nice idea, but my understanding was that use of that form is discouraged - see this post for example: http://www.cakesolutions.net/teamblogs/understanding-akkas-recommended-practice-for-actor-creation-in-scala